sql/parser/sql.y

```diff
@@ -336,7 +338,7 @@ import "strings"
 %token <str>   FALSE FAMILY FETCH FILTER FIRST FLOAT FOLLOWING FOR
 %token <str>   FORCE FOREIGN FORWARD FREEZE FROM FULL FUNCTION FUNCTIONS
 
-%token <str>   GLOBAL GRANT GRANTED GREATEST GROUP GROUPING
+%token <str>   GLOBAL GRANT GRANTED GRANTS GREATEST GROUP GROUPING
```

Comment:
the first list it can go into without causing a reduce conflict is ```type_func_name_keyword```. I have to say my knowledge of yacc is about 15 years old, so I don't remember how it builds the parser, just following the comments around ```unreserved_keywords```